backend_new/src/services/user.service.ts

```diff
@@ -24,12 +41,21 @@ const view: Prisma.UserAccountsInclude = {
   userRoles: true,
 };
 
+type findByOptions = {
+  userId?: string;
+  email?: string;
+};
+
 @Injectable()
 export class UserService {
-  constructor(private prisma: PrismaService) {}
+  constructor(private prisma: PrismaService) {
+    dayjs.extend(advancedFormat);
```

Comment:
Should we add this advancedFormat extension at the app level rather than at the services that need them?

Reply:
As of right now only the user service uses this extension (in both the prisma and typeorm backend) so I think we are okay doing this at the app level and then if we find a use case later on in another service we can make this app wide